### PR TITLE
add capacitor

### PR DESCRIPTION
**Pull request by madsnedergaard**
Add capacitor in a separate dir, building of the apk / archive still has to be done in android studio / xcode.

Generated icons and splash screens but they are not very good so we will have to do them through android studio and xcode

### CODE REVIEW

**Review by Kongkille — COMMENTED** [review]
Is there some kind of README where we can see how to run it and use it?

**Comment by tonypls** [comment]
> Is there some kind of README where we can see how to run it and use it?

Coming right up !!!

File: mobileapp/capacitor.config.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { CapacitorConfig } from '@capacitor/cli';
+
+let config: CapacitorConfig;
+
+const baseConfig: CapacitorConfig = {
+  appId: 'com.tmrow.electricitymap',
+  appName: 'Electricity Maps',
+  webDir: '../webz/dist',
```
Comment (madsnedergaard) [inline mobileapp/capacitor.config.ts:8]:
```suggestion
  webDir: '../web/dist',
```

**Comment by tonypls** [comment]
> Is there some kind of README where we can see how to run it and use it?

Ahh I had global ignored readme files 😆 

It's there now

File: mobileapp/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "mobileappz",
```
Comment (Kongkille) [inline mobileapp/package.json:2]:
```suggestion
  "name": "mobileapp",
```

Hugely important

**Review by Kongkille — APPROVED** [review]
This is cool, I haven't tried it yet though but it might be best to just get it in?

**Pull request merged**